src/v2/Apps/Consign/Routes/SubmissionFlow/ThankYou/ThankYou.tsx

```diff
@@ -24,19 +26,40 @@ export const ThankYou: React.FC = () => {
 
   return (
     <>
-      <Text variant="xxl" mt={4}>
-        Thank you for submitting a work
-      </Text>
-      <Box maxWidth="720px" mt={4}>
-        <Text variant="lg">
-          We’ll email you within 1–3 business days to let you know the status of
-          your submission.
-        </Text>
-        <Text variant="lg" mt={2}>
-          In the meantime, feel free to submit another work—and benefit from
-          Artsy’s low fees, informed pricing, and multiple selling options.
-        </Text>
-      </Box>
+      {isSWAMCEnabled ? (
```

Comment:
@dariakoko we should also be checking if the user is logged in since non logged in users have no collection

Reply:
I don't think so. If the user is not logged in, the FF check is returning null

Reply:
I'll double check

Reply:
yeah, that's true actually. I didn't think about that. No action is needed then. I would recommend though for future work to check if the user is signed in or not explicitly. That's because once we would like to remove this feature flag, we will have to be careful about the user login status